student.rb

```diff
@@ -3,8 +3,8 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(age:, name: 'Unknown', parent_permission: true)
-    super(name: name, age: age, parent_permission: parent_permission)
+  def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] again as the previous CR mentioned this will not work, please remove the colons, if you have any questions or you don't understand the requirements please leave a message in the PR.